srcpkgs/python3-slugify/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'python3-slugify'
+pkgname=python3-slugify
+# Please don't update. Only required by Transifx-client,
+# doesn't work with newer versions
```

Comment:
To help, add an 'update' file that ignores all updates aboive 1.2.6

Reply:
Right, thanks